synapse_data_warehouse/synapse/dynamic_tables/R__populate_usergroup_latest.sql

```diff
@@ -0,0 +1,21 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE OR REPLACE DYNAMIC TABLE USERGROUP_LATEST
```

Comment:
Can we see if there is `CREATE OR ALTER` command for dynamic tables?

Reply:
As the [doc](https://docs.snowflake.com/en/sql-reference/sql/create-table#label-create-or-alter-table-usage-notes:~:text=Currently%20only%20supports%20permanent%2C%20temporary%2C%20and%20transient%20tables.%20Read%2Donly%2C%20external%2C%20dynamic%2C%20Apache%20Iceberg%E2%84%A2%2C%20and%20hybrid%20tables%20are%20not%20supported.): `CREATE OR ALTER` currently only supports permanent, temporary, and transient tables. Read-only, external, dynamic, Apache Iceberg™, and hybrid tables are not supported.